pkg/deployment/inventory.go

```diff
@@ -92,23 +92,14 @@ func GenerateNodeSetInventory(ctx context.Context, helper *helper.Helper,
 					//host.Vars[string.Join(entry, "_gateway_ip")] = res.Gateway
 					host.Vars[entry+"_host_routes"] = res.Routes
 					dnsSearchDomains = append(dnsSearchDomains, res.DNSDomain)
+					host.Vars["dns_search_domains"] = dnsSearchDomains
 				}
-
-				host.Vars["ansible_user"] = getAnsibleUser(instance, nodeName)
-				host.Vars["ansible_port"] = getAnsiblePort(instance, nodeName)
-				host.Vars["management_network"] = getAnsibleManagementNetwork(instance, nodeName)
-				host.Vars["networks"] = getAnsibleNetworks(instance, nodeName)
-
-				ansibleVarsData, err := getAnsibleVars(helper, instance, nodeName)
```

Comment:
Does this still provide as much value now that we're using a single inventory file from the nodeSet?

Group vars should apply to the member nodes, so duplicating all values onto the node itself seems redundant?

Reply:
I think we need this for the cases where we need to set some specific variable for a specific node

Reply:
This would take care of the individual node overrides though?

https://github.com/openstack-k8s-operators/dataplane-operator/pull/431/files#diff-874aae70e738f15dc1bf30750cee0e87e3278c5c02d37e35c0943ca6c29393bbR70